plugins/home/package.json

```diff
@@ -59,6 +59,7 @@
   "peerDependencies": {
     "react": "^16.13.1 || ^17.0.0",
     "react-dom": "^16.13.1 || ^17.0.0",
+    "react-router": "*",
```

Comment:
Do you think this may be possible to remove? No other plugin has it. Maybe possible to at least move to devDependencies as `^6.3.0` since it's only used in tests? Or can MemoryRouter also be imported from `react-router-dom` instead?

Reply:
Thank you for caching this. Let me take a look and I will get back to you.

Reply:
This is fixed now. I've added in the commit relevant to a previous PR: https://github.com/backstage/backstage/pull/19644 (part of this series of PRs)